lib/DDG/Spice/Mo.pm

```diff
@@ -0,0 +1,33 @@
+package DDG::Spice::Mo;
+
+# ABSTRACT: This just creates the carder for Mo's learning modes :)
+
+# Start at http://docs.duckduckhack.com/walkthroughs/forum-lookup.html if you are new
+# to instant answer development
+
+use DDG::Spice;
+use strict;
+use warnings;
+
+# Caching - http://docs.duckduckhack.com/backend-reference/api-reference.html#caching
+spice is_cached => 1;
+spice proxy_cache_valid => '200 1d'; # defaults to this automatically
+
+spice wrap_jsonp_callback => 0; # only enable for non-JSONP APIs (i.e. no &callback= parameter)
+
+# API endpoint - http://docs.duckduckhack.com/walkthroughs/forum-lookup.html#api-endpoint
+spice to => 'https://learn-with-mo.herokuapp.com/duckduckgo?query=$1';
+
+# Triggers - https://duck.co/duckduckhack/spice_triggers
+triggers any => 'learn', 'learn more', 'learn about', 'learn more about';
```

Comment:
I suspect you want `triggers start` to handle only queries that start with the word "learn", otherwise this is going to be far too broad of a trigger as any query that contains the word "lean" would trigger this.